test-configs.yaml

```diff
@@ -876,18 +881,33 @@ device_types:
     class: arm-dtb
     boot_method: uboot
 
+  sun7i-a20-cubieboard2:
+    mach: allwinner
+    class: arm-dtb
+    boot_method: uboot
+
   sun8i_a23_evb:
     name: 'sun8i-a23-evb'
     mach: allwinner
     class: arm-dtb
     boot_method: uboot
 
+  sun8i-a33-olinuxino:
```

Comment:
I can't find this device type in upstream LAVA.  Have you submitted it?

Reply:
Oups, i have sent all devicetype to upstream and forgot this one.
Fixed soon:
https://git.lavasoftware.org/lava/lava/merge_requests/661